index.html

```diff
@@ -42,5 +45,169 @@ <h2>LET'S CONNECT</h2>
 </ul>
 </section>
 
+
+<section class="project-container">
+<div class="project">
+    <img src="Images/SnapshootPortfolio.png" alt="SnapshootPortfolio">
+    <h2>Tonic</h2>
+
+<ol>
+    <li>CANOPY</li>
+    <li><img src="Images/Counter.png" alt="Counter"></li>
+    <li>Back End Dev</li>
+    <li><img src="Images/Counter.png" alt="Counter"></li>
+    <li>2015</li>
+</ol>
+
+<p>A daily selection of privately personalized reads;no accounts or sign-ups required.</p>
+
+<ul>
+    <li>html</li>
+    <li>css</li>
+    <li>javaScipt</li>
+</ul>
+<button type="button" class="button"> See Project</button>
+</div>
```

Comment:
- [ ] Please ensure that the spacing between the list of technologies(i.e HTML, CSS, JS) follows that of the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/164947933-5377e229-71e4-4840-bf47-c7deb28eb9d0.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/164947950-0f8850b1-9236-4cfb-bdc0-4e67fb9f27af.png)